src/UE4SSProgram.cpp

```diff
@@ -859,7 +859,7 @@ namespace RC
             {
                 // Create the mod but don't install it yet
                 if (std::filesystem::exists(sub_directory.path() / "scripts")) m_mods.emplace_back(std::make_unique<LuaMod>(*this, sub_directory.path().stem().wstring(), sub_directory.path().wstring()));
-                if (std::filesystem::exists(sub_directory.path() / "dlls")) m_mods.emplace_back(std::make_unique<CppMod>(*this, sub_directory.path().stem().wstring(), sub_directory.path().wstring()));
+                if (cpp_sdk_loaded && std::filesystem::exists(sub_directory.path() / "dlls")) m_mods.emplace_back(std::make_unique<CppMod>(*this, sub_directory.path().stem().wstring(), sub_directory.path().wstring()));
```

Comment:
Not sure if return would be appropriate there as it would break loading any lua mods that come after a cppmod, it would work in a case of there being two different loops for lua/cpp loading, but that would produce some code duplication related to paths.

Reply:
Yeah I totally missed the fact that we're creating LuaMods in the same loop as CppMods, my bad.
It looks fine then.